util/src/main/java/com/google/cloud/hadoop/util/RetryHttpInitializer.java

```diff
@@ -154,7 +160,8 @@ public CredentialOrBackoffResponseHandler() {
       errorCodeHandler.setBackOffRequired(
           response ->
               BASE_HTTP_BACKOFF_REQUIRED.isRequired(response)
-                  || response.getStatusCode() == HTTP_SC_TOO_MANY_REQUESTS);
+                  || response.getStatusCode() == HTTP_SC_TOO_MANY_REQUESTS
+                  || response.getStatusCode() == HTTP_REQUEST_TIMEOUT);
```

Comment:
How about adding a list of retryable http codes ?

Reply:
Now have a Set